tests/integration/test_random.py

```diff
@@ -20,6 +20,8 @@ def test_hash_deterministic_order(cluster):
     """
 
     cluster.create(3)
+    for i in [1, 2, 3]:
+        cluster.node(i).raft_config_set('loglevel', 'debug')
```

Comment:
Don't we already use "debug" level in tests?

Reply:
wasn't sure, wanted to be sure, as this isn't common, but you might be right.  doesn't hurt as is, just redundant at worst.

Reply:
okay